File: ts/test/exists.test.ts

import { test, describe } from 'node:test'
import { equal } from 'node:assert'


import { WeatherDataApi3SDK } from '..'


describe('exists', async () => {

  test('test-mode', async () => {
    const testsdk = await WeatherDataApi3SDK.test()
    equal(null !== testsdk, true)
  })

})

File: lua/test/exists_test.lua
-- WeatherDataApi3 SDK exists test

local sdk = require("weather-data-api3_sdk")

describe("WeatherDataApi3SDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
